internal/project/project.go

```
@@ -0,0 +1,661 @@
+		ID     json.RawMessage `json:"id"`
+		Number json.RawMessage `json:"number"`
+		Owner  json.RawMessage `json:"owner"`
+		Public json.RawMessage `json:"public"`
+		Title  json.RawMessage `json:"title"`
+		URL    json.RawMessage `json:"url"`
+	}
+	if err := decodeJSON(data, &fields); err != nil {
+		return projectJSON{}, err
+	}
+	if len(fields.ID) == 0 || len(fields.Number) == 0 || len(fields.Owner) == 0 || len(fields.Public) == 0 || len(fields.Title) == 0 || len(fields.URL) == 0 {
+		return projectJSON{}, fmt.Errorf("Project response is missing required fields")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)